kernel/arch/dreamcast/include/dc/video.h

```diff
@@ -191,12 +190,13 @@ extern vid_mode_t vid_builtin[DM_MODE_COUNT];
 */
 extern vid_mode_t *vid_mode;
 
-// These point to the current drawing area. If you're not using a multi-buffered
-// mode, that means they do what KOS always used to do (they'll point at the
-// start of VRAM). If you're using a multi-buffered mode, they'll point at the
-// next framebuffer to be displayed. You must use vid_flip for this to work
-// though (if you use vid_set_start, they'll point at the display base, for
-// compatibility's sake).
+/** These point to the current drawing area. If you're not using a multi-buffered
```

Comment:
```suggestion
/*  These point to the current drawing area. If you're not using a multi-buffered
```
I don't think this is meant to be a doxygen comment...

Reply:
Fixed in force push. Whoops, had forgotten that's reserved for doxygen!